### ISSUE TITLE
Dark Mode for validators

### ISSUE THREAD

**Issue by o-fish-wildaid** [WildAid/o-fish-android#382]
This issue depends on #368 being completed. When Dark Mode is on - the validators should look like this: https://mongodb.invisionapp.com/share/PCYZ6ZW7FR8#/screens/433591789_Boarding_-_Crew_-_Validators

**Comment by abdulbasit75** [comment]
Can I work on this?

**Comment by Sheeri** [comment]
Hello @abdulbasit75, thanks for offering to work on this! I have assigned it to you.

**Comment by yoobi** [comment]
Hello, is help still needed on this issue?

**Comment by Sheeri** [comment]
@yoobi yes! this ticket is still open. I have assigned it to you. Thank you for volunteering! (and good to see you again for this year's hacktoberfest!)

**Comment by yoobi** [comment]
haha thank you! I was trying to get my sandbox login/password and when I received the link to reset it, it was a localhost url :/
`http://localhost:3000/#/reset_password?token=XXXX&tokenId=XXXX`

**Comment by yoobi** [comment]
I've created the pull request, and also a pull request for #414
